db/seedfiles/notice_triggers_seed.rb

```diff
@@ -454,6 +454,28 @@
     #   ]
     # },
 
+  {
```

Comment:
d49c70ffbb This commit is causing trigger seed to fail. I removed syntax error which is been causing.